src/test/java/org/reactome/addlinks/test/PharosDataRetrieverIT.java

```diff
@@ -20,9 +49,24 @@
  * @author sshorser
  *
  */
+//@RunWith(PowerMockRunner.class)
+//@PrepareForTest({ EntityUtils.class, HttpEntity.class })
+//@PowerMockIgnore({"jdk.internal.reflect.*", "jdk.xml.internal.*", "javax.xml.*", "org.apache.logging.*", "javax.management.*","javax.net.ssl.*", "org.xml.*"})
+//@PrepareForTest({ org.apache.http.impl.client.HttpClients.class, PharosDataRetriever.class, PharosTargetsDataRetriever.class})
```

Comment:
Can this be removed along with imports for PowerMock if it's not needed?

Reply:
I think you were looking at an earlier commit - it should have been cleaned up in https://github.com/reactome/add-links/pull/162/commits/5e286e05583282507d4a99fcf95137b21c0dfe7b

Reply:
Yeah, I think I was - sorry about that!